src/lambda/activation.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { AwsExplorer } from '../awsexplorer/awsExplorer'
+import { ext } from '../shared/extensionGlobals'
+import { deleteLambda } from './commands/deleteLambda'
+import { invokeLambda } from './commands/invokeLambda'
+import { LambdaFunctionNode } from './explorer/lambdaFunctionNode'
+
+/**
+ * Activates Lambda components.
+ */
+export async function activate(extensionContext: vscode.ExtensionContext, awsExplorer: AwsExplorer): Promise<void> {
```

Comment:
1. where is this activated from? I don't see a change to extension.ts
2. why is importLambda registered in src/awsexplorer/activation.ts but other lambda commands are registered here?
3. are the delete and invoke command registrations removed from elsewhere?